src/App.tsx

```diff
@@ -1,8 +1,14 @@
+import { Text } from "@mantine/core";
+import MenuBar from "./components/MenuBar/MenuBar";
+
 function App() {
   return (
-    <>
-      <h1>Hello World!</h1>
-    </>
+    <div style={{ display: "flex", gap: "20px " }}>
+      <MenuBar></MenuBar>
```

Comment:
```suggestion
      <MenuBar />
```

If you don't have to pass children to the component, maybe use self closing tags? It looks much cleaner.